src/command_modules/azure-cli-resource/HISTORY.rst

```diff
@@ -2,9 +2,13 @@
 
 Release History
 ===============
+2.1.10
+++++++
+* Minor fixes
+
 2.1.9
 +++++
-`group deployment create`: support uri based parameters file 
+* `group deployment create`: support uri based parameters file
```

Comment:
Should this be in 2.1.10 or was it already released?

Reply:
Already release, AFAIK. Adding this asterisk was a minor formatting fix on my part.

Reply:
I just confirmed this. The current formatting is broken on [pypi.org](https://pypi.org/project/azure-cli-resource/), but the actual note was there.